modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -443,6 +444,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyht
 		retries++
 	}
 
+	// number of retries actually performed
+	repl.Set("http.reverse_proxy.lb.retries", retries)
```

Comment:
Retries aren't _really_ part of load balancing, I wonder if we should just call it `http.reverse_proxy.retries`.

Reply:
I have renamed the placeholder to `http.reverse_proxy.retries` and adjusted its description. I will also edit the description of the PR shortly